code/modules/mob/mob.dm

```diff
@@ -78,6 +78,10 @@
 	initialize_actionspeed()
 	update_movespeed(TRUE)
 
+	action_bindings = list()
+	for(var/i in 1 to 10)
+		action_bindings += null
```

Comment:
Please don't create a 10-elements list for every mob, populated on initialization.
Use a lazy associative list instead.

Reply:
non living have actions

Reply:
alright, disregard the living part
still, I'm sure you can do this with a lazy list instead

Reply:
i'm not actually sure if I can, I need the 10 null entries but I'm a bit tired so i might be wrong. Ill take a peek at it.